src/Infrastructure/Form/Regulation/GeneralInfoFormType.php

```diff
@@ -163,14 +168,38 @@ private function getCategoryOptions(): array
         ];
     }
 
+    private function getVisaModels(array $visaModels = []): array
+    {
+        $choices = [
+            'regulation.general_info.visa.placeholder' => '',
+            'DiaLog' => [],
+        ];
+
+        foreach ($visaModels as $visaModel) {
+            if ($visaModel->organizationUuid) {
+                $choices[$visaModel->organizationName][$visaModel->name] = $visaModel->uuid;
+            } else {
+                $choices['DiaLog'][$visaModel->name] = $visaModel->uuid;
+            }
+        }
+
+        return [
+            'choices' => $choices,
+            'label' => 'regulation.general_info.visa',
+            'help' => 'regulation.general_info.visa.help',
```

Comment:
Refléter le nom du champ pour mieux s'y rertouver

```suggestion
            'label' => 'regulation.general_info.visa_model',
            'help' => 'regulation.general_info.visa_model.help',
```